### PR TITLE
Testbranch PULL REQUEST

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
OLD CODE CHANGES
- add linters check
- add readme
- modified css and html 
- changed background and main picture

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,81 @@
 # TEST-REVIEW
 To try and review the addmission code
+>  project
+-This project is aimed at visiting a pass code and editing it back.
+- Make few necesary changes
+- Review the code your self then submit for further review
+
```
Comment (tufoinnkuo10) [inline README.md:7]: That's a great project description 😄 , But don't you think it gives very little information about the project, some few additional details will do 👍🏾

File: index.html [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title id="title">Dr AndrewTufoin</title>
+  </head>
+  <body>
+    <div class="full-body">
+      <h2>
+        MICROVERSE
+      </h2>
+      <div>
```
Comment (tufoinnkuo10) [inline index.html:15]: The heading tag is perfect but it is the main heading though, so I think h1 will be preferable than  <h2>

File: index.html [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
+        <h4>
+            The tribute page lays emphasis on how microverse assist student who have the zeal<br>
+            and no knowledge about programming to become a remote software developer.
```
Comment (tufoinnkuo10) [inline index.html:18]: Good work, 😄  I think an introduction will have been necessary, microverse needs to be introduced at least don't you think 💯

**Review by tufoinnkuo10 — COMMENTED** [review]
Great work you have there, your improvement is 150% 😆 
but lets perfect the work with this few changes 🛩️ 😃

**Comment by Davidosky007** [comment]
## Hi @tufoinnkuo10 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**